stacker/plan.py

```diff
@@ -82,7 +82,6 @@ def run(self):
             if watcher:
                 stop_watcher.set()
                 watcher.join()
-        return self.ok
```

Comment:
Why was this removed?  Are you sure this isn't used anywhere?  I'm pretty sure the return code is used by the walk method.

Reply:
Right, does this cause any issues though?  Like - if we're going for cleanliness of code, you should probably look through  the call sites to figure out if they need it.  Maybe they don't - and we should probably remove more return statements.  @ejholmes will likely know more, but this goes through the graph and into the threads.

Reply:
@ejholmes do you remember why you return self.ok was this supposed to be self.status ?

Reply:
The original DAG implementation used the Step's return value to determine if any dependencies failed, but that's no longer the case since that's done here:

https://github.com/remind101/stacker/blob/91a571dd85a5474d87b8250851e4ae13071512cf/stacker/plan.py#L368-L376

Removing the return value should be safe (and all tests pass, so).